scripts/metrics/common/logging.py

```diff
@@ -0,0 +1,65 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Standardized logging util file for metrics scripts
+"""
+
+import enum
+import logging
+import sys
+
+
+class Level(enum.IntEnum):
+    CRITICAL = 50
+    ERROR = 40
+    WARNING = 30
+    INFO = 20
+    DEBUG = 10
+    NOTSET = 0
+
+
+class Formatter(logging.Formatter):
+    def __init__(self):
+        super().__init__("%(asctime)s  %(name)-40s  %(message)s")
+
+    def format(self, record):
+        return "{} {}".format(Formatter.get_level_acronym(record.levelno), super().format(record))
+
+    @classmethod
+    def get_level_acronym(cls, level):
+        if level >= Level.CRITICAL:
```

Comment:
Why is ```>=``` comparisons being used here instead of ```==```? Are these level ranges?

Reply:
I don't have a strong reason. I initially thought there might be more logging values for the levels that we didn't include in our enums, but everything seems 1 for 1 so we could probably just check equals to.

Reply:
I would make them ```==``` checks, and default to maybe the ```f"?({level})``` or something like that to indicate its a level that is not matched

Reply:
Fixed